From 2354a83f1a22aab9b0ce480bf3b6df78fb4952b6 Mon Sep 17 00:00:00 2001
From: Alex Wilson <alex.wilson@joyent.com>
Date: Fri, 27 Oct 2017 15:05:19 -0700
Subject: [PATCH] OS-6425 loader should ignore active multi_vdev_crash_dump
 feature on zpool Reviewed by: Patrick Mooney <patrick.mooney@joyent.com>
 Reviewed by: Jerry Jelinek <jerry.jelinek@joyent.com> Approved by: Jerry
 Jelinek <jerry.jelinek@joyent.com>

---
 usr/src/boot/sys/boot/zfs/zfsimpl.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/usr/src/boot/sys/boot/zfs/zfsimpl.c b/usr/src/boot/sys/boot/zfs/zfsimpl.c
index f34f88d88e..26654c0469 100644
--- a/usr/src/boot/sys/boot/zfs/zfsimpl.c
+++ b/usr/src/boot/sys/boot/zfs/zfsimpl.c
@@ -59,6 +59,7 @@ static const char *features_for_read[] = {
 	"org.open-zfs:large_blocks",
 	"org.illumos:sha512",
 	"org.zfsonlinux:large_dnode",
+	"com.joyent:multi_vdev_crash_dump",
 	NULL
 };
 
-- 
2.21.0

